### PR TITLE
*: add empty help_topic table

### PR DESCRIPTION
**Pull request by shenli**
Make mycli happy. Fix: https://github.com/pingcap/tidb/issues/1436
@zimulala @coocood

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: bootstrap.go [hunk 1]
```diff
@@ -182,6 +193,8 @@ func doDDLWorks(s Session) {
 	mustExecute(s, CreateGloablVariablesTable)
 	// Create TiDB table.
 	mustExecute(s, CreateTiDBTable)
+	// Create help table
```
Comment (zimulala) [inline bootstrap.go]: Add . at the end of a line.

**Comment by shenli** [comment]
PTAL @zimulala

**Comment by zimulala** [comment]
LGTM

**Pull request merged**